app/controllers/accounts_controller.rb

```diff
@@ -141,11 +141,7 @@ def tag_requested?
   end
 
   def filtered_status_page(params)
```

Comment:
The `params` parameter isn't used anymore.

Reply:
Oh actually, `params_slice` isn't defined because it's in `Api::BaseController`. I guess it could be moved to `ApplicationController`?

Reply:
The method is small so I just copied it over but it would make sense to maybe put that stuff into a concern instead of relying on an ever more confusing inheritance structure. Wouldn't know what to name the concern though.